src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -964,6 +965,26 @@ public static void TestingStringsConversionToGuid(string testString, string expe
             Assert.Equal(json.BytesConsumed, json.CurrentState.BytesConsumed);
         }
 
+        [Theory]
+        [MemberData(nameof(JsonGuidTestData.ValidGuidTests), MemberType = typeof(JsonGuidTestData))]
+        public static void TryGetGuid_HasValueSequence_RetrievesGuid(string testString, string expectedStr)
```

Comment:
This seems unrelated?

Reply:
yes, I was trying to add code coverage along the way

Reply:
Improving code coverage is good, but ideally the PR only has one purpose.  The GUID tests would be better as a standalone change.

Reply:
Sure. I'm pushing up a PR in one hour to complete the draft, in that I will also remove this test